src/kvirc/ui/KviChannelWindow.cpp

```diff
@@ -104,7 +104,7 @@ KviChannelWindow::KviChannelWindow(KviConsoleWindow * lpConsole, const QString &
 	    this, SLOT(topicSelected(const QString &)));
 	// mode label follows the topic widget
 	m_pModeWidget = new KviModeWidget(m_pTopSplitter, *this, "mode_");
-	KviTalToolTip::add(m_pModeWidget, __tr2qs("Channel modes"));
+	KviTalToolTip::add(m_pModeWidget, __tr2qs("Channel settings button."));
```

Comment:
This is the channel modes text box, not a button. That text is rarely seen anyway unless you're on a channel with no modes.